data/dataset_classes/gnhk.txt

```diff
@@ -0,0 +1,94 @@
+ 
```

Comment:
not sure that it is dataset classes, more looks like model vocabulary file

Reply:
Hi, this file contains the list of characters (the predicted classes) supported by the model. The previous models for Chinese and Japanese also have similar files under `dataset_classes`: [scut_ept.txt](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_classes/scut_ept.txt) and [kondate_nakayosi.txt](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_classes/kondate_nakayosi.txt), respectively.

Reply:
looks like it was wrong decision, because these files have more relation to model, then dataset by itself (somebody can train model with another symblos order or include/exclude symblos on the same datasets and it also does not mean that it works only for these dataset, you can use them for decoding on different data).
But looks like transition should be done later, for all affected models